psa-crypto-sys/README.md

```diff
@@ -29,7 +29,7 @@ makes no difference and there is no way to allow dynamic linking. The
 requirements for configuring and building MbedTLS can be found
 [on their repository homepage](https://github.com/ARMmbed/mbedtls#tool-versions).
 
-When the crate builds the library, both the `mbedcrypto` library
+If the `prefix` feature is enabled, both the `mbedcrypto` library
```

Comment:
Maybe specify what happens if it's not enabled?

Reply:
Added it